### ISSUE TITLE
Getting accessibility errors related to ARC toolkit

### ISSUE THREAD

**Issue by anand24590** [olifolkerd/tabulator#4555]
While running ARC toolkit tool I am getting some accessibility errors related to Tabulator. Which I can't customize and has to be taken care in library code. Please check the list of bugs below:

1. ARIA widget has an incorrect owned element
    Description: The combobox element is not expected inside the grid role

<img width="1186" alt="Screenshot 2024-08-16 at 2 12 13 PM" src="https://github.com/user-attachments/assets/94b76816-57e5-470c-b71e-85c074c62eae">

2. ARIA widget missing required context role
    Description: The rowgroup role is missing the {{requiredContextRole}} required context role

<img width="1195" alt="Screenshot 2024-08-16 at 2 11 27 PM" src="https://github.com/user-attachments/assets/a76bef6a-45e7-4d08-aceb-0a9470d433f9">

3. Non-active element in tab order
    Description: The element has been placed in the Tab order using tabindex="0", but it lacks a specific role=" ..." attribute.

<img width="1223" alt="Screenshot 2024-08-16 at 2 13 15 PM" src="https://github.com/user-attachments/assets/9dc594c2-bbeb-4b0f-abb0-ef0de60bec67">

**Tabulator Info**
- Which version of Tabulator are you using?
- Version: 6.2.1

**To Reproduce**
1. Create table using tabulator with any basic setup

```
{
	data: this.data,
	columns: ["col1", "col2"],
	layout: 'fitColumns',
	textDirection: "ltr",
	reactiveData: true
}
```
5. Run [ARC Toolkit ](https://chromewebstore.google.com/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce?hl=en)
6. and check for errors it received

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] iOS
 - Browser [e.g. chrome, safari] Chrome

**Additional context**
Please let me know if you need any code example. will share it

**Comment by olifolkerd** [comment]
This will be fixed in todays release
